src/container/property-list/property-list.tsx

```diff
@@ -75,3 +103,11 @@ function isPropertyType(type: Types.PatternPropertyType): (p: Model.ElementPrope
 		return typeof patternProperty !== 'undefined' && patternProperty.getType() === type;
 	};
 }
+
+function isGrouped(grouped: boolean): (p: Model.ElementProperty) => boolean {
+	return p => {
+		const patternProperty = p.getPatternProperty();
+		const hasGroup = typeof patternProperty !== 'undefined' && patternProperty.getGroup() !== '';
```

Comment:
Should be `groupName`

Reply:
As discussed, let's name it 'hasGroup' as before